client/src/components/Bookings/BookingCard.tsx

```diff
@@ -0,0 +1,44 @@
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import useStyles from './useStyles';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+import SettingsIcon from '@mui/icons-material/Settings';
+
+interface Props {
+  requestId?: string;
+  userId?: string;
+  start: string;
+  sitterId?: string;
+  status?: string;
+}
+
+const BookingCard = ({ start, status }: Props): JSX.Element => {
+  const classes = useStyles();
+  return (
+    <Card className={classes.bookings}>
+      <Box display="flex" justifyContent="space-between">
+        <CardContent>
+          <Typography className={classes.pos} variant="h3">
+            {start}
+          </Typography>
+        </CardContent>
+        <SettingsIcon fontSize="small" />
+      </Box>
```

Comment:
Optionally, we could apply the styles to `CardContent` and get rid of the `Box` component here - CardContent is meant to be a [basic container](https://github.com/mui-org/material-ui/blob/b3e794791441fa8e6da50ca8545c40d566081f81/packages/material-ui/src/CardContent/CardContent.js#L1-L20) provided mostly for visual clarification (e.g. "card's content starts/ends here"). It works either way so not that important to change anything here 👍🏼